docs/components/profiler/spica/examples/glm5-disagg-pareto-frontier-sweep.md

```
@@ -0,0 +1,162 @@
+---
+# SPDX-FileCopyrightText: Copyright (c) 2025-2026 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+title: GLM-5-FP8 Disaggregated Pareto Sweep
+subtitle: Experimental Spica search for a GLM-5-FP8 Pareto frontier on B200 GPUs
+---
+
+> [!WARNING]
+> **Experimental.** These replay results characterize one software snapshot and workload. Do not
+> treat them as production capacity guidance or a performance commitment. Spica's search behavior
+> and output may change without a standard deprecation period.
+
```

markdownlint-cli2 (0.23.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)